src/bp-core/bp-core-functions.php

```diff
@@ -4920,13 +4920,7 @@ function( $file ) {
 		return array( '12.0' );
 	}
 
-	// Try to get the first major version that was in used when BuddyPress was fist installed.
-	$first_major = '';
-	if ( $initial_version ) {
-		$first_major = bp_get_major_version( $initial_version );
-	}
-
-	$index_first_major = array_search( $first_major, $deprecated_functions_versions, true );
+	$index_first_major = array_search( $initial_version, $deprecated_functions_versions, true );
```

Comment:
Here is another **`$initial_version`** is a **float** value and **`$deprecated_functions_versions`** is an **array of strings**.

Reply:
Thanks for this one too 👍

Reply:
It should be ok now the list of versions is an array of float values.